src/compound/SupplyVault.sol

```diff
@@ -113,6 +118,7 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
     ) internal override {
         _accrueUnclaimedRewards(from);
         _accrueUnclaimedRewards(to);
+        super._beforeTokenTransfer(from, to, 0);
```

Comment:
Passing `0` as the `amount` is weird and may lead to unexpected behaviors down the inheritance tree. Why aren't we passing the third (silenced) argument `amount` to the super's implementation?